app.css

```diff
@@ -0,0 +1,193 @@
+@import url('https://fonts.googleapis.com/css2?family=Work+Sans:wght@500&display=swap');
+
+body {
+    font-family: 'Work Sans';
+    font-style: normal;
+    font-weight: 600;
+    font-size: 30px;
+    line-height: 35px;
+    letter-spacing: 0.1em;
+
+    color: #000000;
+    background: #FFE4C2;
+}
+
+:root{
+    --primary-color: #FFFFFF;
+    --presult-color: #E1EFFF;
+    --phistory-color:#272727;
+    --poutput-color:#747474;
+    --pclear-color:#00DEC3;
+    --pnumber-color: #E1EFFF;
+    --ptext-color: #000000;
+    --poperator-color: #89C0FF;
+
+
+    --secondary-color: #0D0D0D;
+    --sresult-color: #1C1C1C;
+    --shistory-color:#D8D8D8;
+    --soutput-color:#747474;
+    --sclear-color: #00C4AC;
+    --snumber-color: #1C1C1C;
+    --stext-color: #ffffff;
+    --soperator-color: #89C0FF;
+}
+
+.dark-theme{
+    --primary-color: #0D0D0D;
+    --secondary-color: #FFFFFF;
+
+    --presult-color: #1C1C1C;
+    --sresult-color: #E1EFFF;
+
+    --phistory-color:#D8D8D8;
+    --poutput-color:#747474;
+
+    --shistory-color:#272727;
+    --soutput-color:#747474;
+
+    --sclear-color:#00DEC3;
+    --pclear-color: #00C4AC;
+
+    --pnumber-color: #1C1C1C;
+    --snumber-color: #E1EFFF;
+
+    --ptext-color: #ffffff;
+    --stext-color: #000000;
+}
+
+#icon{
+    float: right;
+    margin-right: 100px;
+    padding-top: 20px;
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+}
+
+/* ========================== */
+/*      iPhone13ProMax 1      */
+/* ========================== */
+
+#iPhone13ProMax1{
```

Comment:
Last I remember we discussed on name change?

Reply:
Container name would be better as in our situation I have changed the code I have to update it here and I will update soon with rest of all changes that you have asked me